### PR TITLE
Issue Templates

### PR DESCRIPTION
**Pull request by Gusted**
You can check [here](https://github.com/Gusted/darkreader/issues/new/choose) how they work

As a good Request from https://github.com/darkreader/darkreader/issues/2281#issuecomment-614209675

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/broken-website.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+Give the link to the website
+
+**How to Reproduce**
+Steps to reproduce the behavior:
+An example for this is
+```
+1. Go to x site
+2. Hover over x button
+3. See that isn't chaning colors
+```
```
Comment (FranklinYu) [inline .github/ISSUE_TEMPLATE/broken-website.md:20]: Would “quotation” syntax works better than fenced code block here?

```suggestion
> 1. Go to x site
> 2. Hover over x button
> 3. See that isn't chaning colors
```

**Comment by Gusted** [comment]
Quoted syntax -> Fenced block Good suggestion by @FranklinYu

File: .github/ISSUE_TEMPLATE/broken-website.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+about: Creates a broken website report about Darkreader
+title: "[Broken Website] "
+labels: Broken Website
+assignees: ''
+
+---
+
+<!--
+Thank you for taking the time to report a broken website.
+Please make sure their aren't existing issue's about this broken website.
```
Comment (alexanderby) [inline .github/ISSUE_TEMPLATE/broken-website.md:12]: I'm not an English speaker, but I think it should be `Please make sure there is no existing issue...`.

Comment (Gusted) [reply .github/ISSUE_TEMPLATE/broken-website.md]: Updated the grammarrr

**Pull request merged**